packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.11.0
+
+* **Breaking Change** Changes the Android implementation of the camera plugin from `camera_android`
+  to `camera_android_camerax`, which has full feature parity with `camera_android` except for all
+  of the limitations listed in `README.md`. To continue using `camera_android`, use `camera <0.11.0`.
```

Comment:
Why tell people to use an old version instead of linking to `camera_android`?

Reply:
Good point -- done